app/views/posts/show.html.erb

```diff
@@ -1,8 +1,29 @@
-<% if @post.present? %>
-  
-        <strong><%= @post.title %></strong><br>
-        <%= @post.text %>
-
-<% else %>
-  <p>Post not found</p>
-<% end %>
\ No newline at end of file
+<div class='flex_horizontal'>
+  <div class='flex_horizontal'> 
+  <h1 class= 'post_title'><%= @post.title %></h1>
+  by
+  <%= link_to user_path(@post.author) do %>
+    <%= @post.author.name-%>
+  <% end %>
+  </div>
+  <div>
+    <span> Comments:<%= @post.comments_counter%>, </span>
+    <span> Likes:<%= @post.likes_counter%></span>  
+  </div>
+</div>
+<p><%= @post.text %></p>
+
+<%= form_for(@post.likes.new, url: user_post_likes_path(@post.author, @post)) do |f| %>
+  <%= f.hidden_field :author_id, value: @current_user.id %>
+  <%= f.submit @liked ? "Dislike" : "Like", class: @liked ? 'liked-button' : 'not-liked-button' %>
```

Comment:
**OPTIONAL**- According to the project requirements, you are not required to have a dislike button, please fix this, a user should be able to like a post as many times as possible. Please fix this 🙏 
**OPTIONAL** - Also, I suggest that you include a link that'd lead to the new form page when a user wants to add a new post. This would make it easier for users to add posts in your app